charts/px-monitor/templates/prometheus/prometheus.yaml

```diff
@@ -181,7 +181,6 @@ spec:
     matchLabels:
       k8s-app: pxcentral-prometheus-operator
       app.kubernetes.io/component: pxcentral-prometheus
-{{- include "px-monitor.labels" . | nindent 6 }}
```

Comment:
what will happen to the `run:` label that identifies this?

Reply:
Did not see any component using 
run: px-monitor or run: px-license-server. 
Moreover, the selector.matchlabels are for the deployment to find its own resource like pods.